udf/mrn_udf_highlight_html.cpp

```diff
@@ -35,13 +35,19 @@ MRN_BEGIN_DECLS
 extern mrn::DatabaseManager *mrn_db_manager;
 extern mrn::ContextPool *mrn_context_pool;
 
+enum target_type {
```

Comment:
How about use `enum class` as below?

```c
enum class target_type {
  TEXT = 0,
  HTML
};


```

Reply:
I had fixed this but `enum class` was not supported on centos-7 && mariadb-10.3.
I have reverted the fix...

Reply:
Could you use `CamelCase` and append `Mrn` prefix?

```suggestion
enum MrnTargetType {
```

Reply:
Fixed.